spec/features/managing_suppliers_spec.rb

```diff
@@ -25,4 +26,15 @@
 
     expect(page).to have_content('Enter a supplier name')
   end
+
+  scenario 'editing a supplier with a blank CODA reference value is successful' do
+    visit admin_suppliers_path
+    click_on 'Supplier 2'
+    click_on 'Edit supplier'
+    fill_in 'Name', with: 'Supplier 2 edited'
+    click_button 'Update supplier'
+
+    expect(page).not_to have_content('must start with “C0” and have 4-5 additional numbers, for example: “C012345”')
```

Comment:
I think I'd prefer this expectation to be less detailed, because it's susceptible to being a false negative if we alter the message even slightly. For example, if we change the message to "[...] and have 3-5 additional numbers", the message might be present, but the spec would think it wasn't.

The second expectation is the most relevant one, as it tests for what we want: that the supplier is updated, so there were no errors, so I'd also be ok with removing the `not_to` expectation.

Reply:
I'll take it, thank you!